index.html.in

```diff
@@ -217,22 +217,22 @@
                                   </div>
                               </div>
 
-                              <div class="modal-backdrop" id="fileDropBackdrop" style="display: none; z-index: 1000;">
-                                <div class="well well-lg" style="text-align: center; position: absolute; top: 35%; left: 35%;">
-                                    <h1 style="font-weight: bold;" data-text="Drop_Document">Drop your document</h1>
+                              <div class="modal-backdrop" id="fileDropBackdrop">
+                                <div class="well well-lg" id="modalWell">
+                                    <h1 id="dropDocHeader" data-text="Drop_Document">Drop your document</h1>
                                 </div>
-                                <div class="modal-backdrop" id="fileDropMask" style="opacity: 0; z-index: 10000;"></div>
+                                <div class="modal-backdrop" id="fileDropMask"></div>
                               </div>
 
-                              <div id="docTranslation" style="display: none;">
+                              <div id="docTranslation" class="noDisplay">
```

Comment:
How do you feel about switching `noDisplay` with `hide`? The latter is much more 'Bootstrapy'.